lib/options.js

```diff
@@ -0,0 +1,136 @@
+"use strict";
+
+var path = require("path");
+var merge = require("lodash.merge");
+var deprecate = require("./util/deprecate");
+
+function Options(eyeglass, options, sassArg) {
+  options = getSassOptions.apply(null, arguments);
+
+  Object.keys(options).forEach(function(key) {
+    this[key] = options[key];
+  }.bind(this));
+}
+
+function eyeglassOptionsFromNodeSassArg(arg) {
+  // if it looks like node-sass...
+  if (arg && arg.render && arg.renderSync && arg.info) {
+    // throw a deprecation warning
+    deprecate([
+      "You should no longer pass `sass` directly to Eyeglass. Instead pass it as an option:",
+      "var options = eyeglass(sassOptions, {",
+      "  engines: {",
+      "    sass: require('node-sass')",
+      "  }",
+      "});"
```

Comment:
This warning still has the options passed as separate options.

Also, not sure what the benefit is of passing this as an option, especially in the `engines` namespace.